package/src/animation/functions/index.ts

```diff
@@ -1,2 +1,3 @@
-export { interpolate } from "./interpolate";
-export { interpolateColors, mixColors } from "./interpolateColors";
+export * from "./interpolate";
+export * from "./interpolateColors";
+export * from "./interpolateVector";
```

Comment:
Could we use overloads in interpolate instead of having multiple different versions of interpolate?

Like:

```ts
const interpolate = (
   value: number,
   inputRange: readonly number[],
   outputRange: readonly number[] | Vector[] | Color[],
   options?: Parameters<typeof interpolate>[3]
 ) => 
```

And then test / switch on type when creating the actual interpolation? (Probably do in another PR)